src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource.java

```diff
@@ -260,7 +262,23 @@ protected void retrieve(@CheckForNull SCMSourceCriteria criteria, SCMHeadObserve
                             @CheckForNull SCMHeadEvent<?> event,
                             TaskListener listener) throws IOException, InterruptedException {
         handleRefreshingPRStore(event, listener);
-        getGitSCMSource().accessibleRetrieve(criteria, observer, event, listener);
+        SCMSourceOwner owner = super.getOwner();
```

Comment:
I think there should be a test for this new section of code

Reply:
Added